upgrade/helpers/tasks.py

```diff
@@ -59,14 +59,14 @@ def check_necessary_env_variables_for_upgrade(product):
         failure.append('Product name should be one of {0}.'.format(
             ', '.join(products)))
     # From which version to upgrade
-    supported_vers = ['6.4', '6.3', '6.2', '6.1', '6.0']
+    supported_vers = ['6.5', '6.4', '6.3', '6.2', '6.1', '6.0']
```

Comment:
Supported version is used for from versions I believe and hence we cannot have 6.5 there as of now.

Reply:
Added 6.5 if someone wants to perform snap to snap upgrade.